### PR TITLE
Issue of presence of unused variables resolved

### LINKED ISSUE #71: Remove Unused Variables from Modules
### Occurrences
- [ ] `components/Date/Date.js`
Line 1: 'parse' is defined but never used  no-unused-vars

- [ ] `components/HackathonCard/HackathonCard.js`
Line 1; 'Image' is defined but never used

- [ ] `components/Profile/ProfileTabs.js`
Line 44: 'id' is defined but never used

- [ ] `pages/hackathon/[slug]/teams/[team_id]/index.js`
Line 5: 'ListGroup' is defined but never used  
Line 12: 'execOnce' is defined but never used  

- [ ] `pages/index.js`
Line 2: 'Sorry' is defined but never used

- [ ] `pages/profile/[username].js`
Line 7:  'GrMail' is defined but never used

### Work Required

- Remove all the occurrences of these variables from the given files.

### PR DESCRIPTION
**Pull request by garimavs**
**Fixes**

Fixes #71  by @shubhanshu02 

**Description**

**Checklist:**

-   [ ] I have read the contribution guidelines in the repository.
-   [ ] My branch is up-to-date with upstream branch.
-   [ ] My pull request has a descriptive title (not a vague title like "Update package.json").
-   [ ] My code follows the established code style of the repository.
-   [ ] I have tried running the project locally and verified that there are no visible errors.

### CODE REVIEW

File: pages/hackathon/[slug]/index.js [hunk 1]
```diff
@@ -813,3 +814,4 @@ function Leaderboard({ status, submissions, loading }) {
         </div>
     )
 }
+// line 738
```
Comment (shubhanshu02) [inline pages/hackathon/[slug]/index.js:817]: Remove this line

File: pages/hackathon/[slug]/index.js [hunk 2]
```diff
@@ -84,6 +84,7 @@ export default function Hackathon() {
                             src={"/images/404.svg"}
                             className="mb-3 mt-3"
                             style={{ maxHeight: "30vh" }}
+                            alt="Error 404"
```
Comment (shubhanshu02) [inline pages/hackathon/[slug]/index.js:87]: Not part of this issue

**Review by shubhanshu02 — CHANGES_REQUESTED** [review]
Just a few changed, I'd say. Also, run `npm run format` at the end before committing the changes.

File: pages/hackathon/[slug]/index.js (hunk 1 above)
Comment (garimavs) [reply pages/hackathon/[slug]/index.js]: the requested change is applied

File: pages/hackathon/[slug]/index.js (hunk 2 above)
Comment (garimavs) [reply pages/hackathon/[slug]/index.js]: the requested change is applied

**Review by shubhanshu02 — APPROVED** [review]
LGTM. Thanks @garimavs.

**Pull request merged**